packages/@react-aria/spinbutton/src/useSpinButton.ts

```diff
@@ -56,17 +56,15 @@ export function useSpinButton(
     onIncrementToMax
   } = props;
   const stringFormatter = useLocalizedStringFormatter(intlMessages);
-  const propsRef = useRef(props);
-  propsRef.current = props;
 
-  const clearAsync = () => clearTimeout(_async.current);
+  const clearAsync = useCallback(() => clearTimeout(_async.current), [_async]);
```

Comment:
the useCallbacks in this file are just to help reduce renders, wasn't needed for the strict mode